extensions/wikia/EmbeddableDiscussions/styles/SettingsModal.scss

```diff
@@ -0,0 +1,106 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+
+.embeddable-discussions-settings-modal.modal {
```

Comment:
I'd make it more nested structure. E.g. `embeddable-discussions-settingsmodal-sort-selected` inside `embeddable-discussions-settingsmodal-sort-item`